src/Gateway.php

```diff
@@ -18,7 +18,7 @@ public function getDefaultParameters()
         return array(
             'apiKeyPublic' => '',
             'apiKeySecret' => '',
-            'merchantId'   => '',
+            'merchantId' => '',
```

Comment:
I had an established style where separators are lined up on successive lines. It's a little thing, but I don't see the point of changing things without reason. Was this changed on purpose?

Reply:
Ok, i will fix that back, its the editor that fixed it, as my config for the editor was like that to fix automatically.